packages/backend/src/setup/createRpcTransactionUpdaters.ts

```diff
@@ -59,10 +59,10 @@ function createL2Provider(
 
   let apiKey = ''
   if (projectId === ProjectId('arbitrum')) {
-    apiKey = config.arbitrumAlchemyApiKey
+    apiKey = config.transactionCountSync!.arbitrumAlchemyApiKey
```

Comment:
Not a fan of this. Maybe there is a way for this function to take ConfigWithTransactionCount instead of Config?